### PR TITLE
Moving to Azure DevOps

### PR DESCRIPTION
**Pull request by vijayrkn**

### CODE REVIEW

**Comment by vijayrkn** [comment]
@joeloff - Updated the KoreBuild to 2.1.7-build-20190219.2

@natemcmaster - Does this version need to be pushed anywhere?

**Comment by natemcmaster** [comment]
> Does this version need to be pushed anywhere?

By "this" do you mean KoreBuild? If so, no it should be available on public feeds.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+name: $(Date:yyyyMMdd).$(Rev:rr)
+
+jobs:
+- template: eng/templates/default-build.yml
+  parameters:
+    agentOs: Windows
+    configuration: Release
+    codeSign: true
+    buildArgs: |
```
Comment (natemcmaster) [inline azure-pipelines.yml]: You can remove this parameter since you are not publishing to ProdCon.

File: build.ps1 [hunk 2]
```diff
@@ -167,29 +173,51 @@ if (Test-Path $ConfigFile) {
...
         elseif ($env:USERPROFILE) { Join-Path $env:USERPROFILE '.dotnet'} `
         elseif ($env:HOME) {Join-Path $env:HOME '.dotnet'}`
         else { Join-Path $PSScriptRoot '.dotnet'}
 }
 
-if (!$Channel) { $Channel = 'dev' }
+if (!$Channel) { $Channel = 'master' }
 if (!$ToolsSource) { $ToolsSource = 'https://aspnetcore.blob.core.windows.net/buildtools' }
 
+[string[]] $ProdConArgs = @()
```
Comment (natemcmaster) [inline build.ps1]: Can be removed unless you want to support ProdCon.

File: build.ps1 [hunk 3]
```diff
@@ -26,14 +23,23 @@ The base url where build tools can be downloaded. Overrides the value from the c
...
 
 .PARAMETER ToolsSourceSuffix
 The Suffix to append to the end of the ToolsSource. Useful for query strings in blob stores.
 
-.PARAMETER Arguments
-Arguments to be passed to the command
+.PARAMETER CI
+Sets up CI specific settings and variables.
+
+.PARAMETER PackageVersionPropsUrl
```
Comment (natemcmaster) [inline build.ps1]: Could be removed. Only used by ProdCon.

File: build.sh [hunk 4]
```diff
@@ -1,8 +1,283 @@
...
+    [ ! -z "${config_tools_source:-}" ] && tools_source="$config_tools_source"
+fi
+
+[ -z "${DOTNET_HOME:-}" ] && DOTNET_HOME="$HOME/.dotnet"
+[ -z "$channel" ] && channel='master'
+[ -z "$tools_source" ] && tools_source='https://aspnetcore.blob.core.windows.net/buildtools'
+
+prodcon_args=()
+
+if [ ! -z "$package_version_props_url" ]; then
```
Comment (natemcmaster) [inline build.sh]: Same comment applies here. You don't need this chunk of code for non-prodcon repos.

File: eng/Baseline.xml [hunk 5]
```diff
@@ -0,0 +1,48 @@
+<Baseline Version="2.1.6">
```
Comment (natemcmaster) [inline eng/Baseline.xml]: Delete this file.

File: eng/Baseline.Designer.props [hunk 6]
```diff
@@ -0,0 +1,407 @@
+﻿<!-- Auto generated. Do not edit manually, use eng/tools/BaselineGenerator/ to recreate. -->
```
Comment (natemcmaster) [inline eng/Baseline.Designer.props]: Delete this file. It doesn't apply to this repo.

File: eng/NuGetPackageVerifier.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (natemcmaster) [inline eng/NuGetPackageVerifier.json]: Delete this file. There is already a NuGetPackageVerifier.json file in the repo root.

File: eng/PatchConfig.props [hunk 8]
```diff
@@ -0,0 +1,15 @@
+<Project>
```
Comment (natemcmaster) [inline eng/PatchConfig.props]: Delete this file.

File: eng/ProdConPublish.csproj [hunk 9]
```diff
@@ -0,0 +1,53 @@
+<Project DefaultTargets="Publish">
```
Comment (natemcmaster) [inline eng/ProdConPublish.csproj]: Delete this file.

File: eng/ProjectReferences.props [hunk 10]
```diff
@@ -0,0 +1,54 @@
+<!-- This file is automatically generated. Run `build.cmd /t:GenerateProjectList` to update. -->
```
Comment (natemcmaster) [inline eng/ProjectReferences.props]: Delete this file.

File: eng/Sources.props [hunk 11]
```diff
@@ -0,0 +1,19 @@
+<Project>
```
Comment (natemcmaster) [inline eng/Sources.props]: Delete this file.

File: eng/scripts/UpdateDependencies.ps1 [hunk 12]
```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env pwsh -c
```
Comment (natemcmaster) [inline eng/scripts/UpdateDependencies.ps1]: Delete.

File: eng/scripts/common.psm1 [hunk 13]
```diff
@@ -0,0 +1,262 @@
+$ErrorActionPreference = 'Stop'
```
Comment (natemcmaster) [inline eng/scripts/common.psm1]: Delete this file.

File: eng/targets/Packaging.targets [hunk 14]
```diff
@@ -0,0 +1,10 @@
+<Project>
```
Comment (natemcmaster) [inline eng/targets/Packaging.targets]: Delete this file.

File: eng/targets/ResolveReferences.targets [hunk 15]
```diff
@@ -0,0 +1,134 @@
+<Project>
```
Comment (natemcmaster) [inline eng/targets/ResolveReferences.targets]: Delete this file.

File: eng/tools/BaselineGenerator/BaselineGenerator.csproj [hunk 16]
```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (natemcmaster) [inline eng/tools/BaselineGenerator/BaselineGenerator.csproj]: Delete this file.

File: eng/tools/BaselineGenerator/Program.cs [hunk 17]
```diff
@@ -0,0 +1,132 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```
Comment (natemcmaster) [inline eng/tools/BaselineGenerator/Program.cs]: Delete this file.

File: eng/tools/BaselineGenerator/README.md [hunk 18]
```diff
@@ -0,0 +1,10 @@
+BaselineGenerator
```
Comment (natemcmaster) [inline eng/tools/BaselineGenerator/README.md]: Delete this file.

File: eng/tools/Directory.Build.targets [hunk 19]
```diff
@@ -0,0 +1,2 @@
+﻿<Project>
```
Comment (natemcmaster) [inline eng/tools/Directory.Build.targets]: Delete this file.

File: eng/tools/Directory.Build.props [hunk 20]
```diff
@@ -0,0 +1,3 @@
+﻿<Project>
```
Comment (natemcmaster) [inline eng/tools/Directory.Build.props]: Delete this file.

File: eng/tools/tools.sln [hunk 21]
```diff
@@ -0,0 +1,34 @@
+﻿
```
Comment (natemcmaster) [inline eng/tools/tools.sln]: Delete this file.

File: build.ps1 (hunk 2 above)
Comment (vijayrkn) [reply build.ps1]: Fixed.

**Comment by vijayrkn** [comment]
Thanks for taking a look @natemcmaster

**Pull request merged**